python/ray/ray_constants.py

```diff
@@ -356,3 +356,6 @@ def to_memory_units(memory_bytes, round_up):
 KV_NAMESPACE_FUNCTION_TABLE = b"fun"
 
 LANGUAGE_WORKER_TYPES = ["python", "java", "cpp"]
+
+# ray whitelist config file absolute path.
+RAY_WHITELIST_PATH = os.environ.get("RAY_WHITELIST_PATH", None)
```

Comment:
If this is only used for pickle, let's name it "RAY_PICKLE_WHITELIST_CONFIG_PATH".
If this also used for remote task whitelist, let name it "RAY_SECURERITY_CONFIG_PATH".

Reply:
I rename it to RAY_PICKLE_WHITELIST_CONFIG_PATH